flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/EmbeddedRocksDBStateBackendTest.java

```diff
@@ -210,7 +211,7 @@ public void setupRocksKeyedStateBackend() throws Exception {
         testStreamFactory.setBlockerLatch(blocker);
         testStreamFactory.setWaiterLatch(waiter);
         testStreamFactory.setAfterNumberInvocations(10);
-
+        rocksDBStateUploader = spy(new RocksDBStateUploader(4));
```

Comment:
I think it's better to move `if enableIncrementalCheckpointing` check here from `verifyRocksDBStateUploaderClosed` so that the uploader isn't created unnecessarily.

We could also use `RocksDBOptions.CHECKPOINT_TRANSFER_THREAD_NUM.defaultValue()` instead of `4` here.